api/desecapi/management/commands/check-slaves.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+from socket import gethostbyname
+from time import sleep
+
+from django.conf import settings
+from django.core.mail import get_connection, mail_admins
+from django.core.management import BaseCommand
+from django.utils import timezone
+import dns.message, dns.query, dns.rdatatype
```

Comment:
missing requirement?

Reply:
fix in ac92d3661201870f5069af1ee21518ebb8396e84